testsuite/documentation/optional.md

```diff
@@ -253,12 +253,12 @@ export VIRTHOST_KVM_PASSWORD=therootpwd
 ```
 
 Make sure the image to use for the test virtual machines is located in
-`/var/testsuite-data/disk-image-template.qcow2` on the virtual hosts.
+`/var/testsuite-data/leap-disk-image-template.qcow2` on the virtual hosts.
```

Comment:
you removed the precise file name in sumaform PR, keeping only the directory - do the same here? or revert in sumaform PR?

Reply:
Fixed.